README.rst

```diff
@@ -37,4 +37,4 @@ Requirements
 ------------
 
 * **Python**: 2.7, 3.4+
-* **Backends**: In-memory (development), Postgresql 9.4+ (production)
+* **Backends**: In-memory (development), Postgresql 9.5+ (production)
```

Comment:
This is a breaking change. I suggest that we keep two queries depending on the version of postgresql being used instead of breaking compat